nebula-exchange_spark_2.2/src/main/scala/com/vesoft/nebula/exchange/reader/ServerBaseReader.scala

```diff
@@ -94,6 +82,30 @@ class MySQLReader(override val session: SparkSession, mysqlConfig: MySQLSourceCo
   }
 }
 
+/**
+ * The PostgreSQLReader extends the ServerBaseReader
+ *
+ * @param session
+ * @param postgreConfig
+ */
+class PostgreSQLReader(override val session: SparkSession, postgreConfig: PostgreSQLSourceConfigEntry)
+    extends ServerBaseReader(session, postgreConfig.sentence) {
+  override def read(): DataFrame = {
+    val url =
+      s"jdbc:postgresql://${postgreConfig.host}:${postgreConfig.port}/${postgreConfig.database}"
+    val df = session.read
+      .format("jdbc")
+      .option("driver", "org.postgresql.Driver")
+      .option("url", url)
+      .option("dbtable", postgreConfig.table)
+      .option("user", postgreConfig.user)
+      .option("password", postgreConfig.password)
+      .load()
+    df.createOrReplaceTempView(postgreConfig.table)
+    session.sql(sentence)
```

Comment:
in  configs, if user does not config sentence, default sentence value is "", which will throw `park.sql.catalyst.parser.ParseException` when execute `session.sql(sentence)`

So, please add check for `session.sql`:
```
if("".equals(sentence)) 
     session.sql(sentence)
```

Reply:
What if I check it when user set the configs ?

```
require(
    host.trim.length != 0 && 
    port > 0 && 
    database.trim.length > 0 && 
    table.trim.length > 0 && 
    user.trim.length > 0 &&
    sentence.trim.length > 0
)
```

It will save many `if`

Reply:
It's contradictory with the default sentence value:
```
getOrElse(config, "sentence", "")
```

1. for default sentence value, I prefer to use `null` but not ""
2. for users, they are allowed to not config the sentence, so we still need to process the two situations for sentence.

Reply:
Setting default as `select * from ${table}` might be better.

Reply:
> If sentence is not configured, executing `select * from ${table}` might be better.

No need to execute `select * from table`, default read from db for sparksql is reading all data.